### PR TITLE
[IMP] theme_*: adapt `s_cover`, `s_banner`, `s_text_cover`

### PR DESCRIPTION
**Pull request by chgo-odoo**
- Requires : https://github.com/odoo/odoo/pull/172725

task-3665300

### CODE REVIEW

File: theme_enark/views/snippets/s_cover.xml [hunk 1]
```diff
@@ -1,10 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-<odoo>
-
-<template id="s_cover" inherit_id="website.s_cover">
```
Comment (stefanorigano) [inline theme_enark/views/snippets/s_cover.xml:4]: Let's not remove templates for now unless really necessary.
We'll have to review each theme, chances are that we'll have to re-create the very same template but applying other customizations ;)
Replace the xpath with something that makes sense for this theme... adjust the padding maybe?

File: theme_anelusia/views/snippets/s_text_cover.xml [hunk 2]
```diff
@@ -4,7 +4,7 @@
 <template id="s_text_cover" inherit_id="website.s_text_cover">
```
Comment (stefanorigano) [inline theme_anelusia/views/snippets/s_text_cover.xml:4]: Looks broken
image

File: theme_beauty/views/snippets/s_cover.xml [hunk 3]
```diff
@@ -4,7 +4,7 @@
 <template id="s_cover" inherit_id="website.s_cover">
     <!-- Section -->
     <xpath expr="//section" position="attributes">
-        <attribute name="class" add="o_full_screen_height bg-black-15 o_cc o_cc5" remove="bg-black-50" separator=" "/>
+        <attribute name="class" add="o_full_screen_height bg-black-15 o_cc o_cc5" remove="bg-black-50 o_cc o_cc4" separator=" "/>
```
Comment (stefanorigano) [inline theme_beauty/views/snippets/s_cover.xml:7]: You're both adding and removing the `o_cc` class ... lucky the adding operation is processed after the removal, so the class it's there, but let's fix that ;)

```suggestion
        <attribute name="class" add="o_full_screen_height bg-black-15 o_cc5" remove="bg-black-50 o_cc4" separator=" "/>
```

File: theme_buzzy/views/new_page_template.xml [hunk 4]
```diff
@@ -165,12 +165,6 @@
     </xpath>
 </template>
 
-<template id="new_page_template_about_s_text_cover" inherit_id="website.new_page_template_about_s_text_cover">
```
Comment (stefanorigano) [inline theme_buzzy/views/new_page_template.xml:168]: Here again, let's not remove templates for now.
Replace the xpath with something that makes sense for this snippet.

File: theme_enark/__manifest__.py [hunk 5]
```diff
@@ -13,7 +13,6 @@
         'views/image_library.xml',
 
         'views/snippets/s_banner.xml',
-        'views/snippets/s_cover.xml',
```
Comment (stefanorigano) [inline theme_enark/__manifest__.py:16]: Chances are that we'll have to add it back, for now let's not remove the file ;)

File: theme_graphene/views/customizations.xml [hunk 6]
```diff
@@ -4,7 +4,7 @@
 <!-- ======== COVER ======== -->
 <template id="s_cover" inherit_id="website.s_cover" name="Graphene s_cover">
```
Comment (stefanorigano) [inline theme_graphene/views/customizations.xml:5]: Text is not readable

image

File: theme_kiddo/views/snippets/s_cover.xml [hunk 7]
```diff
@@ -4,7 +4,7 @@
 <template id="s_cover" inherit_id="website.s_cover">
     <!-- Section -->
     <xpath expr="//section" position="attributes">
-        <attribute name="class" add="o_full_screen_height o_cc o_cc1" remove="bg-black-50" separator=" "/>
+        <attribute name="class" add="o_full_screen_height o_cc1" remove="bg-black-50 o_cc4" separator=" "/>
```
Comment (stefanorigano) [inline theme_kiddo/views/snippets/s_cover.xml:7]: Buttons are not very visible

| this pr | master |
|--------|--------|
| image | image |

File: theme_loftspace/views/snippets/s_cover.xml [hunk 8]
```diff
@@ -4,11 +4,11 @@
 <template id="s_cover" inherit_id="website.s_cover">
     <!-- Section -->
     <xpath expr="//section" position="attributes">
-        <attribute name="class" add="o_full_screen_height pt232" remove="pt96" separator=" "/>
+        <attribute name="class" add="o_full_screen_height" separator=" "/>
```
Comment (stefanorigano) [inline theme_loftspace/views/snippets/s_cover.xml:7]: Unreadable text

image

File: theme_monglia/views/customizations.xml [hunk 9]
```diff
@@ -33,7 +33,7 @@
  <template id="s_cover" inherit_id="website.s_cover" name="Monglia s_cover">
```
Comment (stefanorigano) [inline theme_monglia/views/customizations.xml:33]: Primary buttons work better here ;)

| this PR | master |
|--------|--------|
| image | image |

**Review by stefanorigano — CHANGES_REQUESTED** [review]
Good @chgo-odoo , some remarks for you, courage 💪 

edit: @chgo-odoo , don't forget commits and PRs titles/descriptions

**Comment by chgo-odoo** [comment]
@stefanorigano changes done, could you check please ?

**Review by stefanorigano — APPROVED** [review]
LGTM, well done 💪

**Comment by bso-odoo** [comment]
@robodoo rebase-ff r+

**Pull request closed without merging**